assignment-1/main.go

```diff
@@ -0,0 +1,56 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"time"
+
+	externalSnapshotClient "github.com/kubernetes-csi/external-snapshotter/client/v6/clientset/versioned"
+	snapshotClient "github.com/shahpratikr/k8s-dev-assignments/assignment-1/pkg/client/clientset/versioned"
+	snapshotInformer "github.com/shahpratikr/k8s-dev-assignments/assignment-1/pkg/client/informers/externalversions"
+	"github.com/shahpratikr/k8s-dev-assignments/assignment-1/pkg/controller"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/rest"
+	"k8s.io/client-go/tools/clientcmd"
+	"k8s.io/client-go/util/homedir"
```

Comment:
Best practice is to group the imports in 3 groups - 
1. Built-in pkgs
2. External pkgs
3. Internal pkgs
```suggestion
	"flag"
	"fmt"
	"time"

	externalSnapshotClient "github.com/kubernetes-csi/external-snapshotter/client/v6/clientset/versioned"
	"k8s.io/client-go/kubernetes"
	"k8s.io/client-go/rest"
	"k8s.io/client-go/tools/clientcmd"
	"k8s.io/client-go/util/homedir"
	
	snapshotClient "github.com/shahpratikr/k8s-dev-assignments/assignment-1/pkg/client/clientset/versioned"
	snapshotInformer "github.com/shahpratikr/k8s-dev-assignments/assignment-1/pkg/client/informers/externalversions"
	"github.com/shahpratikr/k8s-dev-assignments/assignment-1/pkg/controller"
```